compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -702,98 +687,158 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<void>
-        new_bias->quantparam(std::move(quantparam));
+    auto bias = dynamic_cast<luci::CircleConst *>(node->bias());
+    if (not bias)
+      return;
 
-        set_bias(output, new_bias);
-      }
-      else
-      {
-        auto input_q = input->quantparam();
-        assert(input_q);
-        assert(input_q->scale.size() == 1); // Only support per-layer quant
-        auto input_scale = input_q->scale[0];
-
-        auto weight_q = weight->quantparam();
-        assert(weight_q);
-        assert(weight_q->scale.size() == 1); // Only support per-layer quant
-        auto weight_scale = weight_q->scale[0];
-
-        float scaling_factor{0};
-        int64_t zp{0};
-        new_bias =
-          asym_quant_bias_per_layer(const_bias, input_scale, weight_scale, &scaling_factor, &zp);
-        auto quantparam = std::make_unique<CircleQuantParam>();
-        quantparam->scale.push_back(scaling_factor);
-        quantparam->zerop.push_back(zp);
-        assert(new_bias->quantparam() == nullptr); // bias should not be quantized before
-        new_bias->quantparam(std::move(quantparam));
-
-        set_bias(output, new_bias);
-      }
-    }
+    if (is_quantized(bias))
+      return;
+
+    auto i = loco::must_cast<luci::CircleNode *>(node->input());
+    auto w = loco::must_cast<luci::CircleNode *>(node->filter());
+    auto new_bias = quantized_bias(i, w, bias);
+    node->bias(new_bias);
+  }
+
+  void visit(luci::CircleDepthwiseConv2D *node)
+  {
+    LOGGER(l);
+    INFO(l) << "QuantizeBias visit node: " << node->name() << std::endl;
+
+    auto bias = dynamic_cast<luci::CircleConst *>(node->bias());
+    if (not bias)
+      return;
+
+    if (is_quantized(bias))
+      return;
+
+    auto i = loco::must_cast<luci::CircleNode *>(node->input());
+    auto w = loco::must_cast<luci::CircleNode *>(node->filter());
+    auto new_bias = quantized_bias(i, w, bias);
+    node->bias(new_bias);
   }
+
+  void visit(luci::CircleTransposeConv *node)
+  {
+    LOGGER(l);
+    INFO(l) << "QuantizeBias visit node: " << node->name() << std::endl;
+
+    auto bias = dynamic_cast<luci::CircleConst *>(node->bias());
+    if (not bias)
+      return;
+
+    if (is_quantized(bias))
+      return;
+
+    auto i = loco::must_cast<luci::CircleNode *>(node->outBackprop());
+    auto w = loco::must_cast<luci::CircleNode *>(node->filter());
+    auto new_bias = quantized_bias(i, w, bias);
+    node->bias(new_bias);
+  }
+
+  void visit(luci::CircleFullyConnected *node)
+  {
+    LOGGER(l);
+    INFO(l) << "QuantizeBias visit node: " << node->name() << std::endl;
+
+    auto bias = dynamic_cast<luci::CircleConst *>(node->bias());
+    if (not bias)
+      return;
+
+    if (is_quantized(bias))
+      return;
+
+    auto i = loco::must_cast<luci::CircleNode *>(node->input());
+    auto w = loco::must_cast<luci::CircleNode *>(node->weights());
+    auto new_bias = quantized_bias(i, w, bias);
```

Comment:
If we apply https://github.com/Samsung/ONE/pull/8540/files#r817393019, this codes may be reduced as following :)

```suggestion
    auto new_bias = quantized_bias(node);
```